pom.xml

```diff
@@ -62,4 +62,32 @@
       </plugins>
     </pluginManagement>
   </build>
+
+  <profiles>
+    <profile>
+      <id>jacoco-report</id>
```

Comment:
Can we align this with our parent POM?
https://github.com/devonfw/maven-parent/blob/master/pom.xml#L350